terraform/application/application.tf

```diff
@@ -69,11 +69,36 @@ module "worker_application" {
 
   docker_image = var.docker_image
 
-  command       = ["bundle", "exec", "rake", "jobs:work"]
+  command       = ["/bin/sh", "-c", "QUEUES=default,low_priority,participant_outcomes bundle exec rake jobs:work"]
```

Comment:
Worth adding a comment in the delayed_job initializer pointing to this? I'm just thinking if someone adds a queue in the future they would expect it to be picked up by default and not realise they'll need to add it here too